resources/site-packages/quasar/navigation.py

```diff
@@ -165,6 +165,11 @@ def run(url_suffix=""):
         if item["label2"][0:8] == "LOCALIZE":
             item["label2"] = getLocalizedLabel(item["label2"])
 
+        # Convert labels to unicode
```

Comment:
There's no use at all in converting `label2`, sorry if I wasn't clear enough, but just wrap `item['label']` in `unicode()` on line 181, keep it simple.